src/pages/leeszaal/[slug].tsx

```diff
@@ -79,19 +157,73 @@ const ReadingRoomPage: NextPage = () => {
 				</div>
 			</section>
 
-			<section className="u-py-48">
+			{/* Start debug */}
+			<section className="u-mt-80">
 				<div className="l-container">
-					{media.length > 0 ? (
-						<MediaCardList items={media} view={mode} />
-					) : (
-						<Placeholder
-							img="/images/lightbulb.svg"
-							title="Start je zoektocht!"
-							description="Zoek op trefwoorden, jaartallen, aanbieders… en start je research."
-						/>
-					)}
+					<Button
+						variants={['sm']}
+						onClick={() => {
+							setQuery({});
+							setSearched(false);
+						}}
+					>
+						Reset
+					</Button>
+					&nbsp;
+					<Button
+						variants={['sm']}
+						onClick={() => {
+							setMedia([]);
+						}}
+					>
+						Hide all items
+					</Button>
+					&nbsp;
+					<Button
+						variants={['sm']}
+						onClick={() => {
+							async function fetchMedia() {
+								const data = (
+									await mock(
+										{ view: 'grid' },
+										query.start,
+										READING_ROOM_ITEM_COUNT
+									)
+								).items;
+								data && setMedia(data);
+								setSearched(true);
+							}
+
+							fetchMedia();
+						}}
+					>
+						Load new data
+					</Button>
 				</div>
 			</section>
+			{/* End debug */}
+
+			<section className="u-py-48">
+				<div className="l-container">{renderMediaCardList()}</div>
+			</section>
+
+			{media.length > 0 && (
```

Comment:
In principe kan je hier `media.length` van maken ipv `media.length > 0`. Doet hetzelfde